styles/index.css

```diff
@@ -1 +1,100 @@
 @import url("./something.css");
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 100%;
+  max-width: 1180px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.Header {
+}
+
+.Header-in {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 85px;
+}
+
+.Header-menu {
+  display: flex;
+  align-items: center;
+}
+
+.Header-left {
+}
+
+.Header-logo {
+}
+.Header-logo img {
+  height: 51px;
+  object-fit: contain;
+}
+
+.Header-right {
```

Comment:
Here it's mentioned 'Header-right' but the formatting is just general, 
```
display: flex;
align-items: center;
```
no elements on the right, which is a bit odd.

Reply:
We divided the header as left and right. There is log on the left, menu and button on the right. This code is done to align menu and button. I don't quite understand what's weird.

Reply:
It's better to use another name, more general. Because you can reuse it somewhere else.